vmngclient/session.py

```diff
@@ -113,13 +113,16 @@ class Session:
     """
 
     def __init__(
-        self, url: str,
+        self,
+        url: str,
         username: str,
         password: str,
         port: Optional[int],
         subdomain: str = None,
         timeout: int = 30
     ):
+        self.url = url
```

Comment:
if it became instance attribute how about using also in other places by accessing with instance attribute?
e.g. in line 126
?

Reply:
Naming will  be fixed in separate PR, created issue for it (issue #38 )